picasa.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://en.wikipedia.org/wiki/Picasa",
+    "version": "3.9.141.Build-259",
+    "description": "Picasa is a discontinued image organizer and image viewer for organizing and editing digital photos, plus an integrated photo-sharing website by Google",
+    "license": "Freeware",
+    "hash": "482c1a547d8d3aa25ee446d30ea986de63ef8c8d68b8d1109dd3d9b714e73e08",
+    "url": "https://github.com/raisercostin/raiser-scoop-bucket/releases/download/picasa-3.9.141.259/picasa39-setup-3.9.141.259.exe#dl.7z",
+    "bin": [
+      ["picasa3.exe","picasa"],
+      ["picasa3.exe","picasa3"],
+      "picasaPhotoViewer.exe"
+    ],
+    "shortcuts": [
+        [
+            "Picasa3.exe",
+            "picasa"
```

Comment:
Shortcuts does not look good with first small letter.

No vendors use this capitalization.

Also one shortcut is enough. (Why there is second totally same?)

Reply:
agree. fixed

Reply:
Mark conversations as resolved.